wallet-new/docs/RelatingWalletSpecToCardano.md

```diff
@@ -989,6 +991,72 @@ data TxMeta = TxMeta {
 ```
 [TxMeta](https://github.com/input-output-hk/cardano-sl/blob/6659d8501c727714a7861ad2e527a337e0a11b86/wallet-new/src/Cardano/Wallet/Kernel/DB/TxMeta/Types.hs#L71-L108)
 
+#### TxMeta creation date
+
+While transaction metadata is considered static data, the `_txMetaCreationAt` field is a special case:
+
+* a new transaction is created (`newPending` and `newForeign`) with creation time set to the _current time_
+* if a pending transaction is later confirmed during `applyBlock`, we disregard the original creation date of the transaction and instead use the "time of the block", which is derived from the block slot
```

Comment:
No, this is not correct. Or, if it is, the code is not correct :)

Reply:
ah, you're right, the code is correct, the bug is in my documentation, let me fix that...